responsibleai/responsibleai/_managers/causal_manager.py

```diff
@@ -182,13 +202,38 @@ def add(
             X_test, alpha=alpha, keep_all_levels=True)
 
         result.policies = []
-        for treatment_feature in treatment_features:
+
+        # Check treatment_cost is valid
+        if isinstance(treatment_cost, int) and treatment_cost == 0:
+            treatment_cost = [0] * len(treatment_features)
+
+        if not isinstance(treatment_cost, list):
+            message = ("treatment_cost must be a list with "
+                       "the same number of elements as "
+                       "treatment_features where each element "
+                       "is either a constant cost of treatment "
+                       "or an array specifying the cost of "
+                       "treatment per sample. "
+                       "Found treatment_cost of type "
+                       f"{type(treatment_cost)}, expected list.")
+            raise ValueError(message)
+        elif len(treatment_cost) != len(treatment_features):
```

Comment:
I there a minimum value for treatment cost? Like zero or something is minimum. Or there is no min max. If there is min and max, then we should check for those and raise UserConfigValidationException.

Reply:
Nope, you can have a negative cost which would mean the default treatment is actually more expensive than the new treatment.

Reply:
Ok. We should ask the maintainers of CausalAnalysis in econml if there are bounds on treatment costs. Sure, you could have any value but there should be some limits.